Tools/autotest/common.py

```diff
@@ -3648,7 +3660,7 @@ def validator(value2, empty=None):
                 return False
         debug_text = "Distance to Location (%.4f, %.4f) " % (loc.lat, loc.lng)
         if target_altitude is not None:
-            debug_text += "at altitude %.1f height_accuracy=%.1f" % (target_altitude, height_accuracy)
+            debug_text += ",at altitude %.1f height_accuracy=%.1f, d" % (target_altitude, height_accuracy)
```

Comment:
What's the `, d` here?!

Reply:
D is short for distance... So it will print d=42.0

The sentence is quite long, with that we got the interesting number in the middle of nowhere... IMHO.